server/public/styles.css

```diff
@@ -0,0 +1,27 @@
+.container {
+  margin:auto;
+}
+
+thead {
+  background-color: rgb(179, 176, 176);
+}
+
+tbody tr:nth-child(odd) {
+  background-color: #f2f2f2;
+}
+
+h1 {
+  margin-top: 10px;
+  margin-left: 20px;
+  margin-bottom: 20px;
```

Comment:
Bootstrap has a lot of spacing helpers you can use.

https://getbootstrap.com/docs/4.5/utilities/spacing/